### PR TITLE
Improvement/upstream

### PR DESCRIPTION
**Pull request by felddy**
# 

## 🗣 Description

Upstream CI improvements.

## 💭 Motivation and Context

Keeping it real.

## 🧪 Testing

All the tests.  

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/secrets/config.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "admin_server": {
+    "api_key_init": "dba840fa0f7e03851ca695801f41c4a50fbbdb653ece0e2cdebbcb2977e5dfe1",
```
Comment (dav3r) [inline src/secrets/config.json:3]: Where did this key come from?

**Review by dav3r — APPROVED** [review]
👍, with one question

File: src/secrets/config.json (hunk 1 above)
Comment (felddy) [reply src/secrets/config.json:3]: I think that is the default key until you change the credentials.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,7 +1,11 @@
 ---
+default_language_version:
```
Comment (jsf9k) [inline .pre-commit-config.yaml:2]: Do we want this backported to the skeletons?

**Review by jsf9k — APPROVED** [review]
Approved.  Please see my one question.

**Review by mcdonnnj — APPROVED** [review]
🛸

File: .pre-commit-config.yaml (hunk 2 above)
Comment (felddy) [reply .pre-commit-config.yaml:2]: I think we do.

I was actually getting into some problems with parse issues with the linters.  Despite being in a python 3 virtual environment some of the linters were pulling in python 2.    

Specifically it was upset about this: https://github.com/cisagov/gophish-docker/blob/4f641a078aa26ab82e8f33efdc513a6eb2a2eeac/tests/container_test.py#L29-L32

**Pull request merged**